src/content/docs/configuration/file-format.mdx

```diff
@@ -110,25 +110,43 @@ the merge queue](/merge-queue/partitions).
 ### Defaults
 
 The `defaults` section provides a means to set configuration values that act as
-fallbacks for actions executed by both [pull request rules](/workflow/) and
-[commands](/commands). Instead of defining the same options repeatedly for
-different rules, you can specify them once in the defaults section. If a
-particular option is directly defined under `pull_request_rules`, it will take
-precedence; otherwise, the system resorts to the values defined in defaults.
+fallbacks for:
+- Actions executed by both [pull request rules](/workflow/) and [commands](/commands)
+- Configuration options for [queue rules](/merge-queue/)
+
+Instead of defining the same options repeatedly for different rules, you can specify
+them once in the defaults section. If a particular option is directly defined under
+`pull_request_rules` or `queue_rules`, it will take precedence; otherwise, the system
+resorts to the values defined in the `defaults` section.
 
 ```yaml
 defaults:
   actions:
     comment:
       bot_account: Autobot
 
+  queue_rule:
+    batch_size: 10
+
 pull_request_rules:
   - name: comment with default
     conditions:
       - label = comment
     actions:
       comment:
         message: I 💙 Mergify
+
+queue_rules:
+  - name: default
```

Comment:
This one has batch too since default is set?

Reply:
This one is the example pre-application of the defaults.